qiskit_experiments/test/iq_pulse_backend.py

```diff
@@ -0,0 +1,255 @@
+        if len(qubits) > 1:
+            QiskitError("TODO multi qubit gates")
+        
+        signal = self.converter.get_signals(schedule_blocks)
+        #schedule = block_to_schedule(schedule_blocks)
+        #signal = self.converter.get_signals(schedule)
+        time_f = schedule_blocks.duration * self.dt
+        result = self.solver.solve(
+            t_span=[0.0, time_f],
+            y0=self.y_0,
+            t_eval=[time_f],
+            signals=signal,
+            method="RK23", #?
+        ).y[0] 
+
+        return result
+
+    def run(self, run_input: Union[QuantumCircuit, List[QuantumCircuit]], **options) -> FakeJob:
+        """run method takes circuits as input and returns FakeJob object with results"""
+
+        self.options.update_options(**options)
+        shots = self.options.get("shots")
+        meas_level = self.options.get("meas_level")
+
+        result = {
+            "backend_name": f"{self.__class__.__name__}",
+            "backend_version": self.backend_version,
+            "qobj_id": 0,
+            "job_id": 0,
+            "success": True,
+            "results": [],
+        }
+
+        if isinstance(run_input, QuantumCircuit):
+            run_input = [run_input]
+
+        experiment_unitaries = copy.deepcopy(self.default_pulse_unitaries)
+
+        for circuit in run_input:
+            for name, schedule in circuit.calibrations.items():
+                for (qubits, params), schedule_block in schedule.items():
+                    if (name, qubits, params) not in experiment_unitaries:
+                        experiment_unitaries[(name, qubits, params)] = self.solve(
+                            schedule_block, qubits
+                        )#schedule_block -> schedule
+
+            psi = self.gound_state.copy()
+            for instruction in circuit.data:
+                qubits, params, inst_name = self._get_info(instruction)
+                if inst_name in ["barrier", "measure"]:
+                    continue
+                unitary = experiment_unitaries[(inst_name, qubits, params)]
+                psi = unitary @ psi
+
+            counts = self._state_vector_to_result(psi / np.linalg.norm(psi), **options)
+            # counts = dict(zip(*np.unique(memory, return_counts=True)))
+            run_result = {
+                "shots": shots,
+                "success": True,
+                "header": {"metadata": circuit.metadata},
+                "meas_level": meas_level,
+                "data": {
+                    "counts": counts,
+                    # "memory": memory,
+                },
+            }
+
+            result["results"].append(run_result)
+        return FakeJob(self, Result.from_dict(result))
+
+
+class SingleTransmonTestBackend(IQPulseBackend):
+    """Three level anharmonic transmon qubit"""
+
+    def __init__(self, omega_01: float, delta: float, lambda_0: float, lambda_1: float):
+
+        omega_02 = 2 * omega_01 + delta
```

Comment:
```suggestion
        omega_02 = 2 * omega_01 + alpha
```
We usually have alpha for the anhrmonicity.

Reply:
done